config/channel/distributed/200-eventing-kafka-configmap.yaml

```diff
@@ -22,9 +22,9 @@ data:
       KeepAlive: 30000000000  # 30 seconds
       MaxOpenRequests: 1 # Set to 1 for use with Idempotent Producer
       TLS:
-        Enable: true
+        Enable: false
       SASL:
-        Enable: true
+        Enable: false
```

Comment:
should we really change the default, for the tests ? 

Look what we have upstream to override some cfg:
https://github.com/knative/eventing/blob/release-0.16/test/config/st-channel-broker.yaml

https://github.com/knative/eventing/blob/release-0.16/test/e2e-common.sh#L171-L183

Reply:
We went back and forth on what the default should be here but after your note and some more discussion, leaving SASL/TLS enabled by default does seem to be a better choice.  Adjusted this and added a sed command to change it for the tests only.